azuracast/liquidsoap_custom.liq

```diff
@@ -383,6 +386,7 @@ radio = switch(track_sensitive=false, transitions=[
     transition_into_show(playlist_jingle_station_id),
     transition_into_show(playlist_jingle_station_id),
     transition_into_show(playlist_jingle_station_id),
+    transition_into_show(playlist_jingle_station_id),
```

Comment:
@baxgas I think this new line should be in the Friday block.

Reply:
@gammaw – the Friday section begins on line 379 and ends on 390 as I see from the comments! 
`379    transition_into_off(playlist_jingle_station_id), #ambient: ^^^ Friday ^^^ 
`
`....`
`390    transition_into_off(playlist_jingle_station_id), #ambient: ^^^ Saturday ^^^
`

Reply:
@gammaw – oh, it is not correct, you were right, it has the comments at the END of the days blocks, I'll fix the position